File: week_1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int spaces, int bricks);

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        print_row(n - i ,i + 1);
    }
}

void print_row(int spaces, int bricks)
{
    for (int i = 1; i < spaces; i++)
    {
        printf(" ");
    }
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("  ");
        for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: week_6/sentimental-mario-less/mario.py
from cs50 import get_int


def main():
    while True:
        height = get_int("Height: ")
        if 0 < height < 9:
            for i in range(height):
                print_hash(height - i - 1, i + 1)
            break


def print_hash(spaces, bricks):
    for s in range(spaces):
        print(" ", end="")

    for b in range(bricks):
        print("#", end="")
    print("")


main()
